.eslintrc

```diff
@@ -0,0 +1,41 @@
+{
+  "parser": "babel-eslint",
```

Comment:
No, I was just curious to know if it was required when working with Babel. Let's keep it.